packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -44,70 +45,30 @@ export const Text = React.forwardRef<HTMLElement, TextProps>(
         [color, textAlignment, truncate, italic, weight, variant, element]
       )
     );
-    const [isOverflowing, setIsOverflowing] = useState(false);
     const internalRef = useRef<HTMLElement>(null);
 
     // wire up the forwarded ref to the internal ref
     useImperativeHandle<HTMLElement | null, HTMLElement | null>(ref, () => internalRef.current);
```

Comment:
`internalRef` and `useImperativeHandle` is no longer needed here. The ref from `React.forwardRef` can be passed directly to `createElement()` and `TruncatedText`. Instead the `internalRef`/`useImperativeHandle` needs to moved and used in `TruncatedText`.